portfolio/src/main/webapp/script.js

```diff
@@ -1,28 +1,37 @@
-// Copyright 2019 Google LLC
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     https://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
 /**
- * Adds a random greeting to the page.
+ * Adds a random fact about myself!
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function genRandomFact() {
+  const facts = [
+    'I love spicey food, but I dislike eating peppers!',
+    'I play the flute!',
+    'I have been taking lots of walks during quarantine',
+    'I am 19 years old!',
+  ];
 
   // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  const fact = facts[Math.floor(Math.random() * facts.length)];
```

Comment:
Consider adding logic so the next fact is always different from the current fact